build.sbt

```diff
@@ -124,6 +125,16 @@ lazy val `loglevels-logback` = project
   )
   .dependsOn(`akka-management`)
 
+lazy val `loglevels-log4j2` = project
+  .in(file("loglevels-log4j2"))
+  .enablePlugins(AutomateHeaderPlugin)
+  .settings(
+    name := "akka-management-loglevels-log4j2",
+    libraryDependencies := Dependencies.LoglevelsLog4j2,
+    mimaPreviousArtifactsSet
+  )
```

Comment:
@RAtioNAn I think this has to use `.disablePlugins(MimaPlugin)` until it has been released for the first time

Reply:
Fixed. Log4j2 version updated as well.